### DISCUSSION TITLE
Add Hide  proposal feature

### DISCUSSION THREAD

**Discussion by Bluecrypt** [snapshot-labs/snapshot#1481] [Ideas]
In the current state, we can only:
- Create a proposal
- Delete a proposal
- Duplicate a proposal

A major functionality that seems missing is:

- **Hide** a proposal:
This is Create a proposal, that can be fully setup as usual, but hidden from the users. Only Admins/Members can see it. 

This is of important benefit for teams, so they can setup proposals ahead of time, but don't want to dilute the voter community' attention towards current active proposals with any upcoming proposals.
The admin/member, can revisit the proposal page, and all they have to do is to publish it.

**Comment by mktcode** [comment]
Some sort of "visibility" feature... yep, legit. 👍🏻
